src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -25,7 +25,6 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "inference_profiler.h"
-
```

Comment:
This line you actually need to keep otherwise the formatter will reorder it with the other includes. The include for this file's include is first and separated by a blank line. All other includes are together.

Reply:
Fixed.